### PR TITLE
[QUAD-235] Fixing the current error and also adding csrf headers

### PR DESCRIPTION
**Pull request by suprajkolluri**
Fixing the current error and also adding csrf headers to all ajax post calls

### CODE REVIEW

File: Quadriga/src/main/webapp/WEB-INF/tiles/layouts/skeleton_right.jsp [hunk 1]
```diff
@@ -12,6 +12,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Quadriga">
     
+    <meta name="_csrf" content="${_csrf.token}"/>
+    <!-- default header name is X-CSRF-TOKEN -->
+    <meta name="_csrf_header" content="${_csrf.headerName}"/>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/tiles/layouts/skeleton_right.jsp]: why do you put the csrf token here?

File: Quadriga/src/main/webapp/WEB-INF/views/conceptcollections/collectiondetails.jsp [hunk 2]
```diff
@@ -67,13 +69,18 @@
 							buttons : {
 								Submit : function() {
 									$(this).dialog("close");
+									var token = $("meta[name='_csrf']").attr("content");
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/conceptcollections/collectiondetails.jsp]: why do you go through metatags and not put the EL expression right hear?

**Comment by suprajkolluri** [comment]
@jdamerow removed meta tags and directly used el to access csrf tokens, Please review

**Comment by jdamerow** [comment]
ok to test

File: Quadriga/src/main/webapp/resources/txt-layout/js/d3networkvisualize.js [hunk 3]
```diff
@@ -10,6 +10,10 @@
 
 
 function d3init(graph, networkId, path,type) {
+	
+    var token = '${_csrf.token}';
+    var header = '${_csrf.headerName}';
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/resources/txt-layout/js/d3networkvisualize.js]: these will not be replace. They are static files that are not run through the jsp engine.

File: Quadriga/src/main/webapp/resources/txt-layout/js/editnetworks.js [hunk 4]
```diff
@@ -28,6 +28,10 @@ var Log = {
 };
 
 function init(json, networkId, path) {
+	
+	var token = '${_csrf.token}';
+	var header = '${_csrf.headerName}';
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/resources/txt-layout/js/editnetworks.js]: same here

File: Quadriga/src/main/webapp/resources/js/handleResult.js [hunk 5]
```diff
@@ -15,6 +15,8 @@ function performAction(obj, id, contextPath) {
     data['resolvedHandlePattern'] = $('#resolvedHandlePattern').val();
     data['handleExample'] = $('#handleExample').val();
     data['resolvedHandleExample'] = $('#resolvedHandleExample').val();
+    var token = '${_csrf.token}';
+    var header = '${_csrf.headerName}';
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/resources/js/handleResult.js]: static file

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Review by jdamerow — APPROVED** [review]

**Pull request merged**